src/main/java/edu/cornell/kfs/sys/aws/batch/AwsSecretValidationStep.java

```diff
@@ -0,0 +1,134 @@
+package edu.cornell.kfs.sys.aws.batch;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.UUID;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.sys.batch.AbstractStep;
+import org.kuali.kfs.sys.context.SpringContext;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+
+import edu.cornell.kfs.sys.aws.AmazonSecretValidationInstance;
+import edu.cornell.kfs.sys.aws.AmazonSecretValidationShared;
+import edu.cornell.kfs.sys.service.AwsSecretService;
+
+public class AwsSecretValidationStep extends AbstractStep {
+    private static final String AWS_SECRET_NAME_VALIDATION_SHARED = "AWS_VALIDATION_SHARED";
+    private static final String AWS_SECRET_NAME_VALIDATION_INSTANCE = "AWS_VALIDATION_INSTANCE";
+
+    private static final String ACCESS_TOKEN = "ACCESS_TOKEN";
+    private static final String REFRESH_TOKEN = "REFRESH_TOKEN";
+    private static final String ACCESS_TOKEN_EXPIRATION_DATE = "ACCESS_TOKEN_EXPIRATION_DATE";
+
+    private static final Logger LOG = LogManager.getLogger();
+
+    protected AwsSecretService awsSecretService;
+
+    @Override
+    public boolean execute(String jobName, Date jobRunDate) throws InterruptedException {
+        LOG.info("execute, starting job");
+        awsSecretService.clearCache();
+
+        try {
+            AmazonSecretValidationShared sharedSpaceSecrets = awsSecretService.getPojoFromAwsSecret(AWS_SECRET_NAME_VALIDATION_SHARED, 
```

Comment:
This variable is not getting used at all after initialization. Is there any code that should be checking the contents or checking for a non-null object?

Reply:
good point, I will add a check